RedditCrap/app/js/popup.js

```diff
@@ -52,6 +52,19 @@ function displaySavedSettings() {
 
         var selectElement = document.getElementById('crappyAction');
         selectElement.selectedIndex = data.crappyAction;
+
+    var radios = document.forms["radio-form"].elements["colour"];
+    var button;
+    for(var i = 0, max = radios.length; i < max; i++) 
```

Comment:
Seems a bit verbose -- `for(var i = 0; i < radios.length; i++)` is the way most people expect it to be written I think.